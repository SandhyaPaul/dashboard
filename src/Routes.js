import React from 'react';
import { Switch, Route } from 'react-router-dom';

import Temp from './login/Temp';
import Login from './login/Login';
import LandingPage from './home-page/LandingPage';

const Routes = () => (
	<Switch>
		<Route exact path="/" component={Temp} />
		<Route path="/login" component={Login} />
		<Route path="/app" component={LandingPage} />
	</Switch>
);

export default Routes;
