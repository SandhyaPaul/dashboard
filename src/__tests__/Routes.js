import React from 'react';
import { shallow } from 'enzyme';
// import { MemoryRouter } from 'react-router-dom';
import toJSON from 'enzyme-to-json';
import Routes from '../Routes';

it('renders button without breaking', () => {
	shallow(<Routes />);
});

it('DOM snapshot of Routes', () => {
	const wrapper = shallow(<Routes />);
	expect(toJSON(wrapper)).toMatchSnapshot();
});
