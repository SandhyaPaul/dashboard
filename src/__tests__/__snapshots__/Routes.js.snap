// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DOM snapshot of Routes 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    path="/login"
  />
  <Route
    component={[Function]}
    path="/app"
  />
</Switch>
`;
